### ISSUE TITLE
Webgl=false doesn't work in studio

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/density#187]
For https://github.com/phetsims/qa/issues/995
When trying to test `?webgl=false` on studio it was unable to load and this error came up.
![nostudio](https://github.com/phetsims/density/assets/41024075/34983a25-1ac3-4896-91ab-6f3961547eac)

**Comment by zepumph** [comment]
Yes thanks. Can you please ignore this as it is already known and not important for this version. Instead can do you some brief testing using `?phetioDebug=false` to silence this error?

**Comment by zepumph** [comment]
Thanks! This is a known problem, and was fixed on main/ with https://github.com/phetsims/joist/commit/00ab6d904e539e1c25634d9cf956d9321c26f4a9 but will not be MR'd into old versions of Hydrogen sims because `?webgl=false` is not a public query parameter, it is just for testing. Closing
